aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -212,6 +212,12 @@ The automatic creation of these files is known as *scaffolding*.
 
 * Open a command window in the project directory (The directory that contains the *Program.cs*, *Startup.cs*, and *.csproj* files).
 
+* On Mac, export the scaffold tool path:
```

Comment:
@ptrm04 Just a very minor item, since this is on the dedicated tab for Visual Studio for Mac, it does not need to be specified again that this is on the Mac.  Thanks very much for contributing this missing step!